docs/basic-docs/atomicdex-api-20-dev/get_raw_transaction.md

```diff
@@ -0,0 +1,63 @@
+# get_raw_transaction
+The  `get_raw_transaction`  method returns hexadecimal encoded signed transactions.
```

Comment:
Slightly more verbose ELI5 version -

`The `get_raw_transaction` method takes a coin and confirmed txid as input, and returns the returns its full signed raw transaction hex.`